packages/react-components/README.md

```diff
@@ -28,3 +28,14 @@ ReactDOM.render(
   document.getElementById('root'),
 );
 ```
+
+### Docs
+
+Docs are hosted at https://aka.ms/fluentui-storybook.
+
+Publishing docs:
```

Comment:
Should we add a step in the release pipeline to do this automatically? (the token can be saved in a variable group)

Reply:
Yes, that's the plan, once we are confident enough with the current workflow and have proper Chromatic account.